models/sessions/scratch/databricks/snowplow_web_sessions_this_run.sql

```diff
@@ -287,6 +344,11 @@ select
         {%- for conv_def in var('snowplow__conversion_events') %}
     {{ snowplow_web.get_conversion_columns(conv_def, names_only = true)}}
         {%- endfor %}
+    {% if var('snowplow__total_all_conversions', false) %}
```

Comment:
Should we include `var('snowplow__conversion_events')` in the conditional `if`?

Reply:
It's already nested within that if block